### ISSUE TITLE
kompakysymykset

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#12145]
**Käsitteen tyyppi**

Muutos olemassa olevaan käsitteeseen

**prefabel**

[kompakysymykset](http://www.yso.fi/onto/yso/p3504)

**Tila**

Käsittelyssä

**Ehdotettu muutos**

Onko wikidata-linkitys ok (eli eng. riddle joke) ja sv. skämtgåta paras mahdollinen vastine (todennäköisesti ei).

**Perustelut ehdotukselle**

Trick question = kuggfråga, ja tämä olisi varmaan parempi tapa kääntää. Ei kai kompakysymys ole välttämättä hauska? Mahdollisesti skämtgåtor + kuggfrågor = kompakysymykset, jos myös "hauskuus" on olennainen osa käsitettä. 

https://www.kielitoimistonsanakirja.fi/#/kompakysymys?searchMode=all

**Ehdottajan organisaatio**

ALLFO

**Comment by tpalonen** [comment]
Joo, hyvä huomio. Pitäisi siirtää Wikipedian puolella Kompakysymys tuohon Trick questionin yhteyteen. Sen jälkeen linkitys siihen.

**Comment by krinay** [comment]
Jos kompakysymyksillä tarkoitettaisiin tietynlaista arvoitustyyppiä, olisi englanninkielinen vastine riddle jokes (tai conundrums, kuten LCSH:ssa, jossa se tosin on vain ohjausterminä termille riddles). 

Mutta ehkä YSO-termillä kompakysymykset on jokin laveampi yleiskielinen merkitys. (YSOssa termeillä arvoitukset ja kompakysymykset on vain RT-suhde, muinaisessa Kula-sanastossa suhde oli hierarkkinen.)

**Comment by tpalonen** [comment]
Itse miellän kompakysymyksen enemmän tuollaisena kuin tuossa englannin Wikipedian Trick question -artikkelissa kuvataan eikä niinkään riddle jokena. Ja tätä minusta puoltavat myös määritelmät Wikipediassa:

Kompakysymys: "Kompakysymys tai vain kompa on kysymys, joka on tarkoituksella tehty niin, että kuulija ymmärtää sen väärin tai kuulija kokee kysymyksen hankalaksi vastata (muuten kuin tietosisällöltään), ikään kuin kompastuu kysymykseen."

Trick question: "A trick question is a question that confuses the person asked. This can be either because it is difficult to answer or because an obvious answer is not a correct one. - - - Other types of trick question contain a word that appears to be irrelevant, but in fact provides a clue."

Riddle joke: "A riddle joke, joke riddle, pseudo-joke or conundrum is a riddle that does not expect the asked person to know the answer, but rather constitutes a set-up to the humorous punch line of the joke."

Näiden määritelmien mukaan sanoisin että kompakysymys = trick question. Itse miellän, että kompakysymys ei välttämättä edellytä punchlinea joka tulee kysymyksen jälkeen vaan koko vitsi/juju voi olla jo kysymyksessä itsessään. Esim. "Kuinka monta munaa tarvitaan neljän munan kakkuun?" Eli siinä missä riddle joke on kysymyksen ja sen kanssa vakiintuneena kulkevan vastauksen muodostama pari, trick questionissä (ja kompakysymyksessä) on lähinnä kyse siitä, että kysymykseen on leivottu jokin hämäys tai pila tai vastaava. Ja kuulija joko huomaa sen tai ei. 

Tästä syystä haluaisin tehdä tuon edellä ehdottamani muutoksen Wikidataan. Mutta en tiedä miten muut tämän kokevat.

**Comment by mariaforsen** [comment]
Eli ymmärtääkseni hauskuus/pila **ei** ole olennainen osa kompakysymyksen määritelmässä (ja oikea eng. on trick question), sen mukaan muutetaan siinä tapauksessa sv. skämtgåtor > kuggfrågor.

Näin käännökset on tehty myös MOTissa: https://www.sanakirja.fi/custom/all/kompakysymys
